acceptance/README.md

```diff
@@ -1,164 +1,439 @@
-These tasks come with some documentation: `rake -T` will give short descriptions, and a `rake -D` will give full descriptions with information on ENV options required and optional for the various tasks.
-
-If you are setting up a new repository for acceptance, you will need to bundle install first.  This step assumes you have ruby and the bundler gem installed.
-
-```sh
-cd /path/to/repo/acceptance
-bundle install --path=.bundle/gems
+## Setup
+Here, you'll get an idea of how to run the acceptance test suite for puppet, as
+well as develop tests. We'll go over a variety of methods for running tests,
+both locally and not.
+
+### Prerequisites
+This assumes you have git, ruby, and the [bundler][] gem installed. You'll need
+a local clone of the puppet repo. All command examples in this readme assume you
+are working in the acceptance directory, where this README is located.
+
+### Installation
+All of the dependencies you need to run and develop tests are defined in
+`Gemfile`. To install them, run `bundle install --path .bundle/gems`. This
+command, as will all the command examples in this README, assume you are working
+in the acceptance directory. If you ever have issues with your runtime
+dependencies, you can update them with `bundle update`, or start over fresh with
+`rm -rf .bundle/gems; bundle install`.
+
+To ensure installation was successful, you can run `bundle exec rake -T`. This
+should return something along these lines:
 ```
+$ bundle exec rake -T
+rake ci:help               # Print usage information
+rake ci:test:aio           # Run the acceptance tests using puppet-agent (AI...
+rake ci:test:gem           # Run the acceptance tests against puppet gem on ...
+rake ci:test:git           # Run the acceptance tests against a git checkout
+rake ci:test:quick         # Run a limited but representative subset of acce...
+rake clean                 # Remove any temporary products
+rake clobber               # Remove any generated files
+```
+To get a detailed description of all of these tasks, run `bundle exec rake -D`.
 
-### Using Git Mirrors
-
-By default if you are installing from source, packages will be installed from Github, from their puppetlabs forks.  This can be selectively overridden for all installed projects, or per project, by setting environment variables.
-
-* SERVER => this will be the address the git server used for all installed projects.  Defaults to 'github.com'.
-* FORK => this will be the fork of the project for all installed projects.  Defaults to 'puppetlabs'.
-
-To customize the server or fork for a specific project use PROJECT_NAME_SERVER and PROJECT_NAME_FORK.
+-------------
 
-For example, run with these options:
+## Running Tests
+### Customizing your Test Targets
+If you are using the vmpooler hypervisor and the internal pooling api, you can
+customize the platforms to test on using the HOSTS environment variable. The
+internal pooling api is only available to Puppet, Inc. employees. You'll set
```

Comment:
How can these tests be run by folks outside of Puppet?

Reply:
By using a hypervisor other than vmpooler, and either testing with git or with nightlies

Reply:
should I have an additional section that highlights that specifically?

Reply:
My thought was more to streamline this for non-Puppet and Puppet folks, so it's clear which approaches will get things started quickest for each. I have a value on quickly starting, with depth available if desired.

Related, this could be an opportunity to use `<details>` + `<summary>` tags to suppress extra detail that might not be relevant to the common user?